model/comments.js

```diff
@@ -0,0 +1,31 @@
+const commentsDatabase = require("./databaseLogic/commentsDatabase");
+const util = require("./Util");
+
+class Comment {
+  constructor(commentsDatabaseClass = commentsDatabase) {
+    this.commentsDatabaseClass = commentsDatabaseClass;
+  }
+
+  async addComment(text, userId, postId) {
+    const newComment = await this.commentsDatabaseClass.addComment(
+      text,
+      userId,
+      // eslint-disable-next-line comma-dangle
+      postId
+    );
+    return newComment;
+  }
+
+  async getCommentsByPostId(postId) {
+    const comments = await this.commentsDatabaseClass.getComments(postId);
+    return comments.map((element) => ({
+      id: element.id,
+      text: element.text,
+      userID: element.user_id,
+      postID: element.post_id,
+      createdDate: util.convertDateToUKFormat(element.created_at),
```

Comment:
I would store a UTC date, and only convert to uk format when displaying. This would also allow you to order results based on the created at date.